css/properties/box-shadow.json

```diff
@@ -218,7 +218,7 @@
               },
               "ie": {
                 "version_added": "9",
-                "notes": "To use <code>box-shadow</code> in Internet Explorer 9 or later, you must set <a href='https://developer.mozilla.org/docs/Web/CSS/border-collapse'><code>border-collapse</code></a> to <code>separate</code>."
+                "notes": "To use <code>box-shadow</code> in Internet Explorer 9 or later, you must set <a href='https://developer.mozilla.org/docs/Web/CSS/border-collapse'><code>border-collapse</code></a> to <code>separate</code>. Also, <code>inset</code must be at the end of the specification. Otherwise the full property will be ignored."
```

Comment:
There should be `"partial_implementation": true` above this line.

Also, change this to:
```json
"notes": [
  "To use <code>box-shadow</code> in Internet Explorer 9 or later, you must set <a href='https://developer.mozilla.org/docs/Web/CSS/border-collapse'><code>border-collapse</code></a> to <code>separate</code>.",
  "<code>inset</code> must be the last keyword in the declaration."
]
```